ui/src/components/helpers/components/BibleVerseAccordion.js

```diff
@@ -26,7 +26,7 @@ export const BibleVerseAccordion = ({ bibleVerse, actionText }) => {
           </Box>
           <AccordionIcon />
         </AccordionButton>
-        <AccordionPanel pb={4}>{passage}</AccordionPanel>
+        <AccordionPanel pb={4} dangerouslySetInnerHTML={{ __html: passage }} />
```

Comment:
why are we using this ya

Reply:
To resolve the line breaks, cuz tiptap editable component doesnt handle line breaks, so need to manually set the inner html